wifiphisher/common/phishinghttp.py

```diff
@@ -19,13 +20,24 @@ def get(self, url):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
```

Comment:
I thought CaptivePortalHandler would be suitable as recently you guys added support for internet access, which would (I assume) neglect the captive portal handler

Also it is no longer a matter of one phishing page, but an entire website.